PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+## Name Of the PR
+
+#### Trello board reference:
```

Comment:
Im not sure of including this section, considering this is a open source project and Trello is very specific to the organization. Lets wait and see what the rest of the guys think. @pMalvasio @CamilaMoscatelli @fernandatoledo @mato2593 @aguscha333 @mjurfest

Reply:
Agree with @glm4. Additionally, mostly reviewers don't have access to other project's trello.

Reply:
Given that this is an open source project I agree that probably this section does not make much sense.

Reply:
fixed. Trello board reference was removed

Reply:
Am I the only one who's still seeing the Trello section?